docs/settings.rst

```diff
@@ -63,3 +63,17 @@ CRAWLERA_BACKOFF_MAX
 Default: ``180``
 
 Max value for exponential backoff as showed in the formula above.
+
+CRAWLERA_FORCE_ENABLE_ON_HTTP_CODES
+------------------------------------
+
+Default: ``[]``
```

Comment:
Haver you considered a more sensible default like `[503, 403]`?

Reply:
I considered but I see two problems with doing it, first it is not backwards compatible, second I don't think it should infer anything even if those are usually sensible defaults I have run into cases where they were good responses and it would be very hard to debug.

Reply:
Right, the backwards compatibility is indeed a problem.